### ISSUE TITLE
Error saving log forwarding configuration

### ISSUE THREAD

**Issue by briand44** [uiowa/uiowa#7284]
# Problem
When setting up log forwarding on uiowa07 an error occurs when saving the configuration.

![Screenshot 2024-01-22 at 9 46 34 AM](https://github.com/uiowa/uiowa/assets/1017022/4dc8cc7f-3726-4023-9641-34d730d7b276)

There is an Acquia ticket related to this. https://acquia.my.site.com/s/case/500Pb0000062JCtIAM/log-forwarding-error

Based on that ticket, it sounds like the secondary load balancer isn't able to connect, likely because of firewall rules. We currently do not have any EIPs for the secondary load balancer so we can't allowlist it.

I tried saving log forwarding config for other applications and saw this same error.

It seems like we are still getting logs forwarded to Splunk from the primary load balancer.

Waiting to hear back from Acquia on next steps.

# Proposed solution

**Comment by joewhitsitt** [comment]
We were told today by Acquia support staff that there is a bug on their end related to this and the balancer error but that it is not preventing successful installation or flow of logging. Do you think we can close this @briand44 ?

**Comment by briand44** [comment]
Yeah, we aren't going to do anything to solve this. If they end up letting us know the secondary balancer IP, we can create a new issue to get that allowlisted.
